packages/ckeditor5-dev-tests/tests/tasks/runautomatedtests.js

```diff
@@ -176,4 +179,41 @@ describe( 'runAutomatedTests', () => {
 				}
 			);
 	} );
+
+	it( 'should warn when the `disallowConsoleUse` option is set to `false`', () => {
```

Comment:
We should also add another unit case, which ensures that `disallowConsoleUse=false` does not add our custom code, so it will be doing a similar assertion to the "should create an entry file before tests execution" case.